Misc/stable_abi.toml

```diff
@@ -1843,6 +1843,10 @@
 [function.PyThread_start_new_thread]
     added = '3.2'
 
+# Not mentioned in PEP 384, was implemented as a macro in Python <= 3.12
+[data.PY_TIMEOUT_MAX]
```

Comment:
@vstinner Could we instead exclude this undocumented constant from the limited API?